Neos.Eel/Tests/Unit/Helper/DateHelperTest.php

```diff
@@ -52,7 +53,7 @@ public function formatExamples()
         return [
             'DateTime object' => [$dateTime, 'Y-m-d H:i:s', '2013-07-03 12:34:56'],
             'timestamp as integer' => [1372856513, 'Y-m-d', '2013-07-03'],
-            'timestamp as string' => ['1372856513', 'Y-m-d', '2013-07-03'],
+            'timestamp as string' => ['@1372856513', 'Y-m-d', '2013-07-03'],
```

Comment:
The change in behaviour here would make this change breaking. Can we make this b/c by checking a `string` argument if it `is_numeric`?